YaST/Repos/generate_ports.sh

```diff
@@ -3,7 +3,7 @@
 # TODO: handle non-OSS repo removal automatically
 
 additionnal_archs="aarch64 armv7hl armv6hl ppc"
```

Comment:
this line can not be true for 15.3, aarch64 and powerpc has the same online repo like x86_64

Reply:
well I didn't use this script :-) but I'll fix it. Thank you!